en/middleware/micrortps.md

```diff
@@ -0,0 +1,338 @@
+# PX4-FastRTPS bridge
+
+
+This bridge adds communication capabilities between a **PX4 Autopilot** and a **Fast RTPS** application through serial ports or
+UDP sockets using **CDR serialization**. The goal is to provide a DDS (Data Distribution Service) interface to PX4. This interface allows sharing information also with the upcoming release of ROS (Robot Operating System), ROS2.
+
+DDS is an standard from the OMG (Object Management Group) providing a real-time pub/sub middleware widely used in aerospace, defense and IoT applications, while in Robotics has been adopted as the middleware for ROS2.
+
+![alt text](res/1_general-white.png)
```

Comment:
This image does not exist. The image should be in the /assets folder. It should also have some clear distinguishing name so that it is obvious that it is rtps related. The "alt text" should also be descriptive.